src/altinn-app-frontend/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "altinn-app-frontend",
-  "version": "3.33.4",
+  "version": "3.33.5",
```

```suggestion
  "version": "3.33.4",
```

No need for a new version here, right?

---

Not really, but I didnt want to overwrite whats currently on CDN, so I figured a new patch version would be better.

I guess this can be removed though, and we can merge some branches with actual new features before this, and we will anyway get a new version out without overwriting the stuff thats already there.